sal-common/src/main/java/org/ow2/proactive/sal/model/NodeProperties.java

```diff
@@ -73,40 +90,61 @@ public void setPrice(Double price) {
     }
 
     public NodeProperties numberOfCores(Integer numberOfCores) {
-        this.numberOfCores = numberOfCores;
+        this.cores = numberOfCores;
         return this;
     }
 
     /**
      * Number of cores the node has.
      * @return numberOfCores
      **/
-    public Integer getNumberOfCores() {
-        return numberOfCores;
+    public Integer getCores() {
+        return cores;
     }
 
-    public void setNumberOfCores(Integer numberOfCores) {
-        this.numberOfCores = numberOfCores;
+    public void setCores(Integer numberOfCores) {
+        this.cores = numberOfCores;
     }
 
-    public NodeProperties memory(Long memory) {
-        this.memory = memory;
+    public NodeProperties ram(Long ram) {
+        this.ram = ram;
         return this;
     }
 
+    public NodeProperties cpuFrequency(Double cpuFrequency) {
+        this.cpuFrequency = cpuFrequency;
+        return this;
+    }
+
+    /**
+     * Sets the CPU frequency in GHz.
+     * @param cpuFrequency CPU frequency in GHz
+     */
+    public void setCpuFrequency(Double cpuFrequency) {
+        this.cpuFrequency = cpuFrequency;
+    }
+
+    /**
+     * Gets the CPU frequency in GHz.
+     * @return cpuFrequency
+     */
+    public Double getCpuFrequency() {
+        return cpuFrequency;
+    }
+
     /**
      * Amount of RAM this node has (in MB).
      * @return memory
      **/
-    public Long getMemory() {
-        return memory;
+    public Long getRam() {
+        return ram;
     }
 
-    public void setMemory(Long memory) {
-        this.memory = memory;
+    public void setRam(Long memory) {
+        this.ram = memory;
```

Comment:
Same. You can do that for every simple getter and setter.
I'll stop adding these kind of comments now.

Reply:
thanks for this, I changed all model classes i was updating in this PR to use Lombok getters, setters and equal and hash. Probably same should be done for other classes in model but will let it for another time :)